operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/reconciler/Constants.java

```diff
@@ -1,6 +1,11 @@
 package io.javaoperatorsdk.operator.api.reconciler;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+
 public final class Constants {
+  // Shared object mapper across app
+  public static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
This is wrong. It shouldn't be shared across the app and always be retrieved from the `ConfigurationService`. The reason for this is that users might want to provide their own mapper and, in the case of the Quarkus extension, actually do.

Reply:
I think I already moved it from a static context, but can take a look.

Reply:
fixed